### ISSUE TITLE
Local path to add-on modules being stored in database?

### ISSUE THREAD

**Issue by naps1saps** [PrestaShop/PrestaShop#12467]
Upgraded 1.7.4 to 1.7.5
Go to modules > update and get a 500 error
Debug: Translation missing
en-US	AdminActions	58	Disable mobile	Disable mobile
en-US	AdminActions	4	Upgrade	                Upgrade
en-US	AdminActions	3	Enable mobile	        Enable mobile

I checked international > languages > modify
Could not locate the above entries in en-US > Admin > Actions

The languages page also says "2 missing" in red but I am unaware which are missing.

**Comment by naps1saps** [comment]
Upgraded 1.7.4 to 1.7.5
Go to modules > update and get a 500 error
Debug:
en-US	AdminActions	58	Disable mobile	Disable mobile
en-US	AdminActions	4	Upgrade	                Upgrade
en-US	AdminActions	3	Enable mobile	        Enable mobile

I checked international > languages > modify
Could not locate the above entries in en-US > Admin > Actions

**Comment by khouloudbelguith** [comment]
Hi @naps1saps,

We need to retrieve the PHP error log and the debug mode report in order to find out what's wrong.
Don't you know how to get this information? Please read the following article:
http://build.prestashop.com/howtos/misc/how-to-create-bug-report/

Thanks!

**Comment by naps1saps** [comment]
Apache error log:
[error.log](https://github.com/PrestaShop/PrestaShop/files/2837317/error.log)

There is no PHP log present

Console log does not show anything on updates page but does on the module page:
main.bundle.js:275 GET https://epicdecals.com/stage/index.php/improve/modules/alerts/count?_token=WqZ8jTJcSSMcn1PX8m4ZrHhOF2ehcXFvBW__Pr8L3IA 500 (Internal Server Error)
send @ main.bundle.js:275
ajax @ main.bundle.js:275
De.(anonymous function) @ main.bundle.js:275
getJSON @ main.bundle.js:275
value @ module.bundle.js?1.7.5.0:3
e @ module.bundle.js?1.7.5.0:3
(anonymous) @ module.bundle.js?1.7.5.0:3
u @ main.bundle.js:275
c @ main.bundle.js:275
setTimeout (async)
(anonymous) @ main.bundle.js:275
u @ main.bundle.js:275
fireWith @ main.bundle.js:275
fire @ main.bundle.js:275
u @ main.bundle.js:275
fireWith @ main.bundle.js:275
ready @ main.bundle.js:275
g @ main.bundle.js:275

**Comment by khouloudbelguith** [comment]
@naps1saps, if you enable the debug mode, there are no exceptions displayed in the module page?
Thanks!

**Comment by khouloudbelguith** [comment]
@naps1saps, in your log file,
`[Wed Feb 06 16:36:16.163280 2019] [php7:warn] [pid 108657] [client 65.158.195.82:30176] PHP Warning:  require_once(/media/data/!WEB/EpicDecals.com/store/modules/paypal/paypal.php): failed to open stream: No such file or directory in /media/web/EpicDecals.com/src/Adapter/Module/Module.php on line 354`
could you please try to rename your paypal folder to paypal_old, into the path Project_Folder/modules/
Thanks to check & feedback.

**Comment by naps1saps** [comment]
I'm not too familiar with the debug thing but as far as I can tell, no.  Turning on debug allows everything to load normally.  Here is a picture.
![untitled](https://user-images.githubusercontent.com/5412452/52359301-7dbcef00-29ee-11e9-85e4-25dd578f8cfc.png)

**Comment by khouloudbelguith** [comment]
@naps1saps, Could you please try to clear cache manually by deleting all the folder into var/cache (prod & dev).
Thanks to check and feedback.

**Comment by naps1saps** [comment]
I uninstalled the paypal module after seeing that.
Checked the log again and EiCaptcha did the same thing
Uninstalled EiCaptcha and now the update page is loading normally.
I feel slightly dumb.  Thank you for sending me in the right direction.

On another note, if installing a module and then changing the storage folder location, it appears it saves the full local path in the database so it is invalid after the move.  Why is that?
